python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -1799,44 +1799,44 @@ def test_torch_tensor_nccl_all_reduce_scheduling(ray_start_regular):
     assert result[2] == (value, shape, dtype)
 
 
-# @pytest.mark.parametrize("ray_start_regular", [{"num_cpus": 4}], indirect=True)
-# def test_torch_tensor_nccl_all_reduce_scheduling_one_ready(ray_start_regular):
-#     """
-#     Test that scheduling avoids deadlocks when allreduce is used.
-#     """
-#     if not USE_GPU:
-#         pytest.skip("NCCL tests require GPUs")
+@pytest.mark.parametrize("ray_start_regular", [{"num_cpus": 4}], indirect=True)
+def test_torch_tensor_nccl_all_reduce_scheduling_one_ready(ray_start_regular):
+    """
+    Test that scheduling avoids deadlocks when allreduce is used.
```

Comment:
Update comments to explain the potential deadlock.

Reply:
done